lib/java/com/google/android/material/chip/ChipDrawable.java

```diff
@@ -1652,6 +1682,10 @@ public void setChipIconResource(@DrawableRes int id) {
     setChipIcon(AppCompatResources.getDrawable(context, id));
   }
 
+  public void setChipIconUseDrawableSize(boolean useDrawableSize) {
```

Comment:
Please add javadoc since the height behavior is not self explanatory.

Reply:
Added javadoc.